### DISCUSSION TITLE
[Feature request] KYC Passport for Celo

### DISCUSSION THREAD

**Discussion by arthurgousset** [celo-org/social-connect#227] [Feature requests (Needs)]
Reflecting this [request from EdRodrigues on Discord](https://discord.com/channels/600834479145353243/786788580944642058/971860571337146389):

> Hey friends. I was thinking... Is there a project being built on Celo around avoiding the re-work of doing KYC of the users again for companies that are integrated with the Fiat Connect API?
> 
> For example: I use Valora in country A, I have to do KYC to use providers there. If I go to country B, there are different providers. Should I do KYC again?
> 
> Or... If I receive a grant from Celo Foundation, I have to do KYC. If I receive another grant from CCF, I have to do KYC again...
>
> What if there was a Celo Passport ?

**Comment by JarekSyg** [comment]
Hey! We ( https://authologic.com ) are working on that currently - let me know if you are interested in learning more about the details.

**Comment by nembal** [comment]
Hey @0xarthurxyz
If the question was referreing to just reusable kyc, then @kycdao is live on CELO soon! :) 
you can read about it on www.kycdao.xyz
